src/olympia/users/models.py

```diff
@@ -399,17 +399,47 @@ def has_anonymous_display_name(self):
         return not self.display_name and self.has_anonymous_username
 
     @cached_property
-    def reviews(self):
-        """All reviews that are not dev replies."""
-        qs = self._ratings_all.filter(reply_to=None)
-        # Force the query to occur immediately. Several
-        # reviews-related tests hang if this isn't done.
-        return qs
-
-    def delete(self, hard=False):
+    def ratings(self):
+        """All ratings that are not dev replies."""
+        return self._ratings_all.filter(reply_to=None)
+
+    def delete_related_content(self):
+        """Delete content produced by this user."""
+        self.collections.all().delete()
+        self.addons.all().delete()
```

Comment:
Eeek, not sure about this - there may be other owners of the add-on.

Reply:
Actually... I misread the issue which calls for *disabling* and not *deleting*. I'll fix.